cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,49 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import ContactFormPageObject
+  from '../support/pages/contactForm.pageObject';
+import HomeAndCataloguePageObject
+  from '../support/pages/homeСatalogue.pageObject';
+import checkoutPageObject
+  from '../support/pages/checkout.pageObject';
+
+
+import faker from 'faker';
 
+const contactForm = new ContactFormPageObject();
+const homePage = new HomeAndCataloguePageObject();
+const checkout = new checkoutPageObject();
+
+const testData = {
+  name: faker.name.firstName(),
+  country: faker.address.country(),
+  city: faker.address.city(),
+  creditCard: faker.finance.creditCardNumber(),
+  month: faker.date.month(),
+  year: faker.random.number({ min: 2018, max: 2028 }),
+};
+
+describe('Demoblaze checkout', () => {
+  before(() => {
+    homePage.visit();
   });
 
-  it('', () => {
+  it('should provide an ability to add product to cart and place order', () => {
+    checkout.clickOnCategory('notebook');
```

Comment:
The 'notebook' string passed to the clickOnCategory function seems to be hardcoded. It would be more flexible to use a variable or constant.